crowbar_framework/app/helpers/backup_helper.rb

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module BackupHelper
+  def date_parse(datestring)
+    DateTime.strptime(datestring, "%Y%m%d-%H%M%S")
+  end
+
+  def delete_backup_button(backup)
+    button_to t("delete"), { action: "delete", name: backup.name, created_at: backup.created_at }, method: "delete", data: { confirm: t("are_you_sure") }, class: "button btn btn-danger pull-right"
+  end
+
+  def restore_button(backup)
+    button_to t(".restore"), restore_path(name: backup.name, created_at: backup.created_at), method: :post, data: { confirm: t(".restore_warning") }, class: "button btn btn-default"
```

Comment:
This can be fixed really easy, just add more brackets and put it in multiple lines.